README.md

```diff
@@ -87,10 +87,11 @@ service:
 
 | parameter    | description                                           | required | default                  |
 | ------------ | ----------------------------------------------------- | -------- | ------------------------ |
+| accessToken  | ArgoCD Access Token                                   | `false`  |                          |
 | action       | Action to perform (upsert, delete)                    | `true`   | upsert                   |
 | annotations  | Key/Value pair of annotations for the application     | `false`  | {}                       |
-| clientId     | ArgoCD Client Id / Username                           | `true`   |                          |
-| clientSecret | ArgoCD Client Secret / Password                       | `true`   |                          |
+| clientId     | ArgoCD Client Id / Username - If no accessToken       | `false`  |                          |
+| clientSecret | ArgoCD Client Secret / Password - If no accessToken   | `false`  |                          |
```

Comment:
Add these comments to `action.yml`: https://github.com/ejhayes/action-argocd-deploy/blob/next/action.yml#L4

Those comments will be updated here as part of the release